elements/eos/payg/libpeas-1.bst

```diff
@@ -1,3 +1,5 @@
+# Derived from gnome-build-meta.bst 48.3 core-deps/libpeas-1.bst
+
```

Comment:
There isn't a libpeas-1.bst in gnome-build-meta. GNOME OS only has libpeas v2.

We should ignore everything in the `eos/payg` directory as everything there is kludges specific to the PAYG boot components.